jwst/straylight/straylight.py

```diff
@@ -20,19 +20,52 @@
 
 # C version of the fitting code
 def makemodel_ccode(fimg, xvec, imin, imax, lor_fwhm, lor_amp, g_fwhm, g_dx, g1_amp, g2_amp):
+    """
+    C code version determining the cross-artifact model to subtract.
 
+    Parameters
+    ----------
+    fimg : ndarray
+        Detector flux values mapped to a single 1D array.
+    xvec : ndarray
+        Array of x pixel values across the detector.
+    imin : int
+        Starting column to fit (1/2 detector at a time)
+    imax : int
+        Ending column to fit
+    lor_fwhm : ndarray
+        Parameter of Lorenztian for each detector row.
```

Comment:
I'm unfamiliar with this step, but I would have thought that this represents the FWHM of the Lorentzian, if this is right can you update the docstring to reflect that?